ui/src/components/config_section/ConfigSectionPanelTitleWithToggle.js

```diff
@@ -0,0 +1,18 @@
+import React, { Fragment } from "react";
+import { EuiFlexGroup, EuiFlexItem, EuiHorizontalRule, EuiTitle } from "@elastic/eui";
+
+export const ConfigSectionPanelTitleWithToggle = ({ title, toggle }) => (
```

Comment:
I think this component can be removed and an `append` prop can be added to  `ConfigSectionPanelTitle` directly to take in the React component and render it if it exists.